orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -428,30 +463,56 @@ Status PaddingElimination::ApplyImpl(Graph& graph, bool& modified, int graph_lev
   NodeArg* squeeze_out_arg = InsertNodesForValidIndices(
       graph, reshape_output_args[0], embedding_node->MutableInputDefs()[2], embedding_node->GetExecutionProviderType());
 
+  std::vector<int64_t> first_two_indices{0, 1};
```

Comment:
use `CreateInitializerFromVector` to create int64_t typed initializers.

Reply:
done